src/pages/dataset/DataSetPageHeader.js

```diff
@@ -44,7 +44,11 @@ export default function DataSetPageHeader({ emailAddress, hasError, dataSet }) {
 
   if (is_processing) {
     return (
-      <DataSetProcessing emailAddress={emailAddress} dataSetId={dataSetId} />
+      <DataSetProcessing
+        emailAddress={emailAddress}
+        dataSetId={dataSetId}
```

Comment:
Is there a reason for having this prop? It doesn't look like the `DataSetProcessing` function uses it, but am I missing it being used elsewhere?

Reply:
good catch i was just removing it